pwndbg/pdb.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This module serves as utility for the developers to get pdb debugging session right, whenever standard:
+    import pdb; pdb.set_trace()
+Can't do its job properly (that happens often as gdb does some magic with stdout/stderr).
+
+Thanks to Dmoreno: http://stackoverflow.com/questions/17074177/how-to-debug-python-cli-that-takes-stdin
+
+Usage:
+    import pwndbg.pdb
+
+Importing the module will print out pretty message on what to do next.
+"""
+import functools
+import pdb
```

Comment:
Probably would be nice.  The issue is that currently we already wrap all invoked `Command` objects (see `commands/__init__.py`) so anything you reach interactively *should* already be wrapped.  Anything you reach non-interactively (i.e. from the `pi` interpreter) does not need to be wrapped.  Clearly, things don't work as they're supposed to.